src/OMSimulatorLib/Model.cpp

```diff
@@ -143,6 +144,19 @@ oms_status_enu_t oms::Model::rename(const ComRef& cref, const ComRef& newCref)
   return logError("Model \"" + std::string(getCref()) + "\" does not contain system \"" + std::string(front) + "\"");
 }
 
+oms::Variable* oms::Model::getVariable(const ComRef& cref) const
+{
+  // renaming any subcomponent
+  oms::ComRef tail(cref);
+  oms::ComRef front = tail.pop_front();
+
+  if (system->getCref() == front)
+    return system->getVariable(tail);
+
+  logWarning("failed to find signal \"" + std::string(cref) + "\"");
```

Comment:
This is just for debugging and should be removed before merging the pull request.